Script/test-podspec.sh

```diff
@@ -28,12 +28,11 @@ echo "
 platform :ios, '10.0'
 target '$PROJECT_NAME' do
   use_frameworks!
-  pod 'AEPCore', '~> 3.0.0'
-  pod 'AEPIdentity', '~> 3.0.0'
-  pod 'AEPLifecycle', '~> 3.0.0'
-  pod 'AEPServices', '~> 3.0.0'
-  pod 'AEPSignal', '~> 3.0.0'
+  pod 'AEPCore', '~> 3.0'
+  pod 'AEPIdentity', '~> 3.0'
+  pod 'AEPServices', '~> 3.0'
   pod 'AEPRulesEngine', '~> 1.0.0'
+  pod 'AEPEdge', '~> 1.0.0'
```

Comment:
Should we use `'AEPEdge', '~>1.1'` as Edge 1.1 is dependent on Edge Identity?

Reply:
Yeah we can do that, for that test it should affect one way or the other given there are no compile-time dependencies between the extensions.

Reply:
For this upcoming release, we should probably keep it at 1.0.0 otherwise we can't release Identity edge until Edge 1.1.0 has been released and propagated through the pods CDN.